zips/zip-0230.rst

```diff
@@ -306,8 +301,13 @@ The OrchardZSA Action Group Description is encoded in a transaction as an instan
 | 64 × ``nActionsOrchard``    |``vSpendAuthSigsOrchard``     |``byte[64 * nActionsOrchard]``                  |Authorizing signatures for each Action of the Action Group in a      |
 |                             |                              |                                                |transaction.                                                         |
 +-----------------------------+------------------------------+------------------------------------------------+---------------------------------------------------------------------+
+| varies                      | ``nAssetBurn``               | ``compactSize``                                |The number of Assets burnt.                                          |
```

Comment:
This change is in the wrong place relative to the other parts of the ActionGroup. Following existing convention, this would be positioned between `nAGExpiryHeight` and `sizeProofsOrchard` (so that the contents of the ActionGroup is effecting data first, authorizing data second).

Reply:
My bad, that's correct. Rearranged this.